src/diart/blocks/diarization.py

```diff
@@ -225,6 +226,9 @@ def __call__(
                 agg_prediction = shifted_agg_prediction
 
             outputs.append((agg_prediction, agg_waveform))
+            if self.return_embeddings:
```

Comment:
I think, we need to have setnroings ducing inference in the real time application. Obviously, every iteration is overkill

> it would be enough to have a way of **extracting them after inference**

My usecase is about global speakers identification in real - time environment, which means I want to obtain speaker embeddings at some points for vector store lookup. After inference is not enough. As for now, I think good idea is to obtain a centroid for each new speaker after it's well - converged. Getting embeddings at each iteration obviously is overkill, but that worka for me at this point, as I can keep only the last state of centroids downstream. Maybe, a better idea is to develop an on-demand method for that, but I am pretty new to real time application and not sure how to do it right